setup.py

```diff
@@ -15,7 +15,7 @@
     'eth-tester[py-evm]>=0.3.0b1,<0.4',
     'web3>=5.2.0,<5.3.0',
     'tox>=3.7,<4',
-    'hypothesis>=4.41.3,<5'
+    'hypothesis[lark]>=4.41.3,<5'
```

Comment:
```suggestion
    'lark>=0.7.8,<1'
    'hypothesis[lark]>=4.41.3,<5'
```

Reply:
Argh; @fubuloubu this is a shitty feature... please don't make commit suggestions if they are going to fail 100%
It's so convenient .. but messes up for the other person.